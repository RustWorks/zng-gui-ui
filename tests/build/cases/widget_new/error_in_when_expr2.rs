use zero_ui::prelude::*;

fn main() {
    let _scope = App::minimal();
    let _ = Wgt! {
        margin = 0;
        when *# {
            margin = 10;
        }
    };
}
