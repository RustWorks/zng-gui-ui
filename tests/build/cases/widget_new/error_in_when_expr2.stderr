error: unexpected end of input, expected identifier
 --> $DIR/error_in_when_expr2.rs:7:18
  |
7 |         when *# {
  |                  ^

warning: unused import: `states::is_pressed`
 --> $DIR/error_in_when_expr2.rs:1:35
  |
1 | use zero_ui::properties::{margin, states::is_pressed};
  |                                   ^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default
